coralproject/cay
# fix infinite redirect on not authorized

## What? #269 

this PR does just what the title says. I moved the auth back into the state.

## How to test

- if you're logged out, the app should redirect to `/login`
- if you're logged in, go to the browser console and type `localStorage.setItem('authorized', false)`. Reload the page and you should be redirected to `/login` instead of being treated to an infinite redirect loop.
